apps/dashboard/lib/trpc/routers/api/setDefaultBytes.ts

```diff
@@ -30,7 +29,7 @@ export const setDefaultApiBytes = t.procedure
             "We were unable to find the KeyAuth. Please try again or contact support@unkey.dev.",
         });
       });
-    if (!keyAuth || keyAuth.workspaceId !== input.workspaceId) {
```

Comment:
why do we make it less secure?

we should never ever ever read a workspaceId from input, only ever from context

Reply:
Am I missing something? Previously we were reading from input now its omitted and just checking if keyAuth exists?